package.json

```
@@ -53,14 +53,14 @@
     "brotli-size": "^0.0.3",
     "camelcase": "^5.0.0",
     "chalk": "^2.4.0",
+    "core-js": "2",
```

```suggestion
    "core-js": "^2.6.0",
```

as people probably already have 2.x.x installed, and this enforces 2.0.0

---

I don't think core-js is really good to be using inside something that bundles since or you are going to opinionate people on their polyfilling or you are going to bundle in polyfills. better to leave it out

---

Removed core-js as it's indeed better to leave out.